### ISSUE TITLE
update.ipfs.io dnslink?

### ISSUE THREAD

**Issue by jbenet** [ipfs/infrastructure#139]
quick q: where is the TXT record set for update.ipfs.io ? i didnt find it in digital ocean

**Comment by jbenet** [comment]
It should now be set to: 

```
/ipfs/QmeEVmRTDX2c9zjPtjHJG6GidrFL7NjqEqP1Q4am6sLfif
```

@whyrusleeping did you add this elsewhere?

**Comment by whyrusleeping** [comment]
its on digitalocean, i'll update it and post the process (maybe i did things weirdly?)

**Comment by whyrusleeping** [comment]
So, on digitalocean, i click the networking tab, then the domains side-tab thing, then i click the magnifying glass next to ipfs.io.

When I made the record it didnt let me make the TXT record as its own entry on the main page.

**Comment by jbenet** [comment]
@whyrusleeping oh i see added it under one domain. i guess that's ok? there's multiple ways to do it because can treat a subdomain as a domain

**Comment by jbenet** [comment]
@whyrusleeping pro tip: use https://github.com/ipfs/dnslink-deploy and something like: https://github.com/ipfs/website/blob/master/Makefile#L30-L32

i think will want:

```
DIGITAL_OCEAN=$(shell cat auth.token) node_modules/.bin/dnslink-deploy \
		--domain=ipfs.io --record=update --path=/ipfs/$(shell cat versions | tail -n1)
```
or 

```sh
> cat publish-dnslink
#!/bin/sh

token="$(cat auth.token)"
path="/ipfs/$(cat versions | tail -n1)"
domain="update.ipfs.io"

echo "publishing $path at $domain"
DIGITAL_OCEAN="$token" dnslink-deploy \
  --domain="$domain" \
  --record="@" \
  --path="$path"
```

**Comment by lgierth** [comment]
@whyrusleeping let me know when ipfs-update uses dist.ipfs.io, then I'll go and delete update.ipfs.io

**Comment by lgierth** [comment]
Closing this. I had deleted the update.ipfs.io records in the meantime, but then it turned out the currently released version of ipfs-update (1.0.0) still uses it -- meh! Reinstated it, and I guess it's there to stay.
